### ISSUE TITLE
Some bundled apps (Protracker 3) are cropped due to NTSC mode

### ISSUE THREAD

**Issue by rsn8887** [amigavision/AmigaVision#213]
Since the WB is loaded with NTSC Screenmode selected, some apps have GUI elements missing on the bottom, such as Protracker 3:

![image](https://github.com/user-attachments/assets/56f01bcb-32b8-4543-82f6-720f5a260d49)

And Mline (although Mline scrolls automatically):

![image](https://github.com/user-attachments/assets/762b34c6-8c5e-42ba-a76b-31d2e8315083)

**Comment by limi** [comment]
Yes, probably needs `setpal` applied, will take a look. (Workbench was always configured for NTSC, but this probably just reads the hardware default when deciding what to render)

**Comment by limi** [comment]
Musicline Editor is fixed, I still need to look at PT3.

**Comment by limi** [comment]
Looks like both of these do the right thing now that the system defaults to PAL in 2025.05.05
